src/main/java/racingcar/StringAddCalculator.java

```diff
@@ -0,0 +1,78 @@
+package racingcar;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    public static int splitAndSum(String inputString) {
```

Comment:
메서드 마다 null or empty string 입력에 대한 처리 로직이 중복으로 사용되는 경우가 있네요.
```java
if (matcher != null) {}
if (inputString != null) {}
if (inputStrings.length == 0) {}
```

splitAndSum 메서드 앞 단에서 한 번만 체크해주면 될 것 같아서 isNullOrBlank() 메서드를 추가해봤습니다. 위에서 언급한 로직은 제거했어요. 테스트는 무사히 통과하네요~
```suggestions
    public static int splitAndSum(String inputString) {
        if (isNullOrBlank(inputString)) {
            return 0;
        }
```

```suggestions
    public static int splitAndSum(String inputString) {
        if (isNullOrBlank(inputString)) {
            return 0;
        }

        String delimiter = getDelimiter(inputString);

        String[] result = splitString(delimiter, inputString);

        return sumStrings(result);
    }